src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -445,7 +447,7 @@ internal void Log(RpcEvent msg)
                 {
                     if (rpcLog.Exception != null)
                     {
-                        var exception = new RpcException(rpcLog.Message, rpcLog.Exception.Message, rpcLog.Exception.StackTrace);
+                        var exception = new Workers.Rpc.RpcException(rpcLog.Message, rpcLog.Exception.Message, rpcLog.Exception.StackTrace);
```

Comment:
Should this change to `GrpcException`? I saw a couple other things where we continued using `Rpc` rather than `Grpc` -- were those missed or is there a reason?

Reply:
Any unhandled exceptions coming from user code for out-of-proc are cast as `RpcException` - Lets leave this for now we can revisit this change as part of #6284